### PR TITLE
22 done-vinz + bonus 7

### PR DESCRIPTION
**Pull request by vinc86**

### CODE REVIEW

File: 22-PB-loops-for/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// task 1
```
Comment (Ungoot) [inline 22-PB-loops-for/index.js:1]: strict

File: 22-PB-loops-for/index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+console.log("Task 2:");
+
+let bottles = ["one", "two", "three", "four", "five"];
+for (let bottle of bottles) {
+
+    if (bottle === "one") {
+        console.log(`There is ${bottle} bottle of beer on the wall.`);
+    } else {
+        console.log(`There are ${bottle} bottles of beer on the wall`)
```
Comment (Ungoot) [inline 22-PB-loops-for/index.js:21]: ;

File: 22-PB-loops-for/index.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+// task 6
+
+console.log("Task 6:");
+
+let sum2 = 0;
+
+for (let i = 0; i <= 1000; i++) {
+    let mult3 = i % 3;
+    let mult5 = i % 5;
+    if (mult3 === 0 && mult5 === 0) {
```
Comment (Ungoot) [inline 22-PB-loops-for/index.js:87]: Should be || not &&

**Comment by vinc86** [comment]
ok, i was not shure about this one

Il giorno mer 3 giu 2020 alle ore 15:04 Bleda <notifications@github.com> ha
scritto:

File: 22-PB-loops-for/index.js (hunk 3 above)
Comment (vinc86) [reply 22-PB-loops-for/index.js:87]: was not shure about this, becouse of the exercise description

**Pull request closed without merging**